openfecli/tests/commands/test_gather.py

```diff
@@ -134,30 +135,29 @@ def test_get_column(val, col):
 solvent	lig_ejm_46	lig_jmc_28	23.3	0.8
 solvent	lig_ejm_46	lig_jmc_28	23.4	0.8
 """
-
-@pytest.fixture()
-def results_dir_serial(tmpdir)->str:
-    """Example output data, with replicates run in serial (3 replicates per results JSON)."""
-    with tmpdir.as_cwd():
-        with resources.files('openfecli.tests.data') as d:
-            tar = tarfile.open(d / 'rbfe_results.tar.gz', mode='r')
-            tar.extractall('.')
-
-        return os.path.abspath(tar.getnames()[0])
-
-@pytest.fixture()
-def results_dir_parallel(tmpdir)->str:
-    """Example output data, with replicates run in serial (3 replicates per results JSON)."""
-    with tmpdir.as_cwd():
-        with resources.files('openfecli.tests.data') as d:
-            tar = tarfile.open(d / 'rbfe_results_parallel.tar.gz', mode='r')
-            tar.extractall('.')
-
-        return os.path.abspath(tar.getnames()[0])
-
-@pytest.mark.parametrize('data_fixture', ['results_dir_serial', 'results_dir_parallel'])
+POOCH_CACHE = pooch.os_cache('openfe')
+ZENODO_RBFE_DATA = pooch.create(
+        path = POOCH_CACHE,
+        base_url="doi:10.5281/zenodo.14884797",
+        registry={
+            "rbfe_results_serial_repeats.tar.gz": "md5:d7c5e04786d03e1280a74639c2981546",
+            "rbfe_results_parallel_repeats.tar.gz": "md5:cc54afe32b56232339a9315f4c3d6d91"},
+    )
+
+@pytest.fixture
+def rbfe_result_dir()->pathlib.Path:
+    def _rbfe_result_dir(dataset)->str:
+        ZENODO_RBFE_DATA.fetch(f'{dataset}.tar.gz', processor=pooch.Untar())
+        cache_dir = pathlib.Path(pooch.os_cache('openfe'))/f'{dataset}.tar.gz.untar/{dataset}/'
+        return  cache_dir
+
+    return _rbfe_result_dir
+
+@pytest.mark.skipif(not os.path.exists(POOCH_CACHE) and not HAS_INTERNET,reason="Internet seems to be unavailable and test data is not cached locally.")
```

Comment:
@mikemhenry this works but misses edge cases where the openfe cache exists, but the test data inside has been deleted (I think this would only happen if a user manually messed with it).

Reply:
I don't think it really misses the edge case, since if someone did that then the test will fail since the data is missing